File: APP TravelEasy/Auth/src/main/java/com/br/TravelEasy/Auth/Feing/Email/EmailMessage.java
package com.br.TravelEasy.Auth.Feing.Email;

import lombok.Getter;
import lombok.Setter;

import java.io.Serializable;
import java.util.List;

@Getter
@Setter
public class EmailMessage implements Serializable {
    private static final long serialVersionUID = 1L;

    private String to;
    private String subject;
    private String body;
    private List<String> attachments;
}

File: APP TravelEasy/EmailSmsSender/src/main/kotlin/com/br/TravelEase/EmailSmsSender/Listener/EmailMessage.kt
package com.br.TravelEase.EmailSmsSender.Listener

import lombok.Getter
import lombok.Setter
import java.io.Serializable

@Getter
@Setter
data class EmailMessage(
    val to: String,
    val subject: String,
    val body: String,
    val attachments: List<String>?
) : Serializable